Project/Assets/Prairie/Framework/Script/Twine/TwineNode.cs

```diff
@@ -31,24 +64,44 @@ public void OnGUI()
 		if (this.enabled && !this.isMinimized) {
 
 			float frameWidth = Screen.width / 3;
-			Rect frame = new Rect (10, 10, frameWidth, Screen.height);
+			Rect frame = new Rect (10, 10, frameWidth, Screen.height / 2);
```

Comment:
Here's a sneak preview of the new resizing:
![](https://media.giphy.com/media/26xBzWUZtbYJvYo2Q/giphy.gif)